package.json

```
@@ -9,8 +9,9 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
```

```suggestion
		"url": "https://sindresorhus.com"
```

unrelated to this PR, but I'm adding the protocol here when cleaning up.